wit-manifest.json

```diff
@@ -13,5 +13,10 @@
         "commit": "50de8a34c19c12de5066cd7ada50ebb5f5b2ea26",
         "name": "rocket-chip",
         "source": "git@github.com:chipsalliance/rocket-chip.git"
+    },
+    {
+        "commit": "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef",
+        "name": "wake-languages-sifive",
+        "source": "git@github.com:sifive/wake-languages-sifive.git"
```

Comment:
We should update this to point at the renamed version of this, which is `wake-languages-sifive.git`. Also, we should make sure to point it at a real commit.

Reply:
fixed